packages/flutter/lib/src/material/list_tile_theme.dart

```diff
@@ -462,6 +480,13 @@ class ListTileTheme extends InheritedTheme {
   /// [ListTileThemeData.enableFeedback] property instead.
   bool? get enableFeedback => _data != null ? _data.enableFeedback : _enableFeedback;
 
+  /// Overrides the default value of [CheckboxListTile.controlAffinity]
+  ///  or [ExpansionTile.controlAffinity] or [SwitchListTile.controlAffinity] or [RadioListTile.controlAffinity]
+  ///
+  /// This property is obsolete: please use the [data]
```

Comment:
Is the reference to 'data' valid here?

Reply:
Yes. I tested it locally for [CheckboxListTile] [ExpansionTile] [SwitchListTile] [RadioListTile] and it works.

```
testWidgets('RadioListTile uses ListTileTheme controlAffinity', (WidgetTester tester) async {
    Widget buildListTile(ListTileControlAffinity controlAffinity) {
      return MaterialApp(
        home: Material(
          child: ListTileTheme(
            // data: ListTileThemeData(
            //   controlAffinity: controlAffinity,
            // ),
            controlAffinity: controlAffinity,
            child: RadioListTile<double>(
              value: 0.5,
              groupValue: 1.0,
              title: const Text('RadioListTile'),
              onChanged: (double? value) {},
            ),
          ),
        ),
      );
    }

    await tester.pumpWidget(buildListTile(ListTileControlAffinity.leading));
    final Finder leading = find.text('RadioListTile');
    final Offset offsetLeading = tester.getTopLeft(leading);
    expect(offsetLeading, const Offset(72.0, 16.0));

    await tester.pumpWidget(buildListTile(ListTileControlAffinity.trailing));
    final Finder trailing = find.text('RadioListTile');
    final Offset offsetTrailing = tester.getTopLeft(trailing);
    expect(offsetTrailing, const Offset(16.0, 16.0));

    await tester.pumpWidget(buildListTile(ListTileControlAffinity.platform));
    final Finder platform = find.text('RadioListTile');
    final Offset offsetPlatform = tester.getTopLeft(platform);
    expect(offsetPlatform, const Offset(72.0, 16.0));
  });
```

Do you think I should continue to add test cases for `data`?

Reply:
Sorry for the confusion: my points was only on the formulation.

```dart

/// This property is obsolete: please use the [data]
/// [ListTileThemeData.controlAffinity] property instead.
```

Does it mean the following? :

```
/// This property is obsolete: please use the
/// [ListTileThemeData.controlAffinity] property instead.
```